sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_quickpulse/_live_metrics.py

```diff
@@ -92,7 +98,8 @@ def __init__(self, **kwargs: Any) -> None:
         id_generator = RandomIdGenerator()
         self._base_monitoring_data_point = MonitoringDataPoint(
             version=_get_sdk_version(),
-            invariant_version=1,
+            # Invariant version 5 indicates filtering is supported
+            invariant_version=5,
```

Comment:
While testing you can update this to 5, but would recommend to keep this at 2 until filtering fully implemented.